gems/pending/VolumeManager/MiqLvm.rb

```diff
@@ -211,12 +211,17 @@ def getPvObj(vgObj, pvName, pv)
     pvObj = PhysicalVolume.new(pv['id'].delete('-'), pvName, pv['device'], pv['dev_size'], pv['pe_start'], pv['pe_count'])
     # Add reference to volume group object to each physical volume object.
     pvObj.vgObj = vgObj
-    # Add reference to the physical volume's open disk object to the physical volume object.
-    dobj = @pvHdrs[pvObj.pvId].diskObj
-    # Add reference to the physical volume object to the physical volume's open disk object.
-    pvObj.diskObj = dobj
+
+    if @pvHdrs.keys.include?(pvObj.pvId)
```

Comment:
Or, since you're accessing it *immediately* afterwards:

```ruby
dobj = @pvHdrs[pvObj.pvId].try(:diskObj)
if dobj
  # Add reference to the physical volume object to the physical volume's open disk object.
  pvObj.diskObj = dobj
  dobj.pvObj = pvObj
end
```